From 9d95326a05e31dcc1ae30fbc1066125faadd7339 Mon Sep 17 00:00:00 2001
From: Slawomir Rosek <slawomir.rosek@cavium.com>
Date: Tue, 27 Nov 2018 13:00:48 +0300
Subject: [PATCH 0784/1051] octeontx-ssow: Fix dependencies in Kconfig

Add missing dependency on SSO PF driver to the SSOW PF Kconfig.

Signed-off-by: Slawomir Rosek <slawomir.rosek@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/cavium/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/cavium/Kconfig b/drivers/net/ethernet/cavium/Kconfig
index 6cf7be85eb77..2246028b85ae 100644
--- a/drivers/net/ethernet/cavium/Kconfig
+++ b/drivers/net/ethernet/cavium/Kconfig
@@ -136,7 +136,7 @@ config OCTEONTX_SSO_PF
 
 config OCTEONTX_SSOW_PF
 	tristate "OcteonTX SSOW physical function driver(SSOW_PF)"
-	depends on 64BIT
+	depends on 64BIT && OCTEONTX_SSO_PF
 	default y
 	help
 	  Select this option to enable SSOW Physical function.
-- 
2.17.1

